cluster/gce/gci/configure-helper.sh

```diff
@@ -182,13 +182,16 @@ function get-or-generate-uuid(){
   fi
 
   # each line of the ssdmap looks like "${device} persistent-uuid"
-  if [[ ! -z $(grep ${device} ${ssdmap}) ]]; then
+  if grep -q "${device}" "${ssdmap}"; then
     #create symlink based on saved uuid
-    local myuuid=$(grep ${device} ${ssdmap} | cut -d ' ' -f 2)
+    local myuuid
+    myuuid=$(grep "${device}" "${ssdmap}" | cut -d ' ' -f 2)
   else
     # generate new uuid and add it to the map
-    local myuuid=$(uuidgen)
-    if [[ ! ${?} -eq 0 ]]; then
+    local myuuid
+    myuuid=$(uuidgen)
+    # shellcheck disable=SC2181
+    if [[ $? -ne 0 ]]; then
```

Comment:
Any reason you disabled the check here instead of using `if ! myuuid=$(uuidgen); then`?

Reply:
No, your suggestion is what should be there. Thank you for this input. Update is coming.